mobile-browser-based-version/README.md

```diff
@@ -1,38 +1,34 @@
 # Disco - Developer Guide
 
-## Code Organisation
-
-    .
-    ├── assets       # contains the resources of the app
-    │ ├── css        # css files
-    │ ├── svg        # svg image / icons
-    ├── components   # vue file folder
-    │ ├── XXX.vue    # vue example file
-    ├── helpers      # typescript helpers folder
-    │ ├── XXX.ts     # js example file
-    ├── platforms    # i18n related folder
-    │ ├── ...
-    ├── router       # vue-router related folder
-    │ ├── index.ts
-    ├── store        # vuex related folder
-    │ ├── store.ts
-    ├── main.ts      # root file of the app
-    ├── .env.XXX     # environment specific variables
-    └── ...          # rest of the files
+Welcome to the Disco developer guide, the guide is broken down into small chapters. To get up and running you can find the information here as well as in the [SERVER](server/README.md) chapter.
+
+If you run into any sort of trouble then hopefully you can find an answer in our [FAQ](information/FAQ.md); otherwise please create a new issue.
+
+To learn more about the development (contributing) process or our architecture you can go to the respective chapters.
+
+#### Chapters
+
+- README _(you are here)_
+- [SERVER](server/README.md)
```

Comment:
helper server, and comes last among the 4

Reply:
yeah, i thought so too, but you also need to read the server readme in order to run the whole thing, whereas the rest is more about info, so i wrote it based on that order. what do u think